src/xcode/ENA/ENA/Source/Services/Exposure Transaction/ExposureDetection.swift

```diff
@@ -27,6 +27,7 @@ final class ExposureDetection {
 	private var completion: Completion?
 	private var progress: Progress?
 	private var countryKeypackageDownloader: CountryKeypackageDownloading
+	private let country = "EUR"
```

Comment:
Should we leave a short(!) comment why this is this fixed "EUR" and not something two-letter coded?

Reply:
Added a comment and linked to the corresponding discussion.